langserver/ast.go

```diff
@@ -59,7 +59,7 @@ func goRangesToLSPLocations(fset *token.FileSet, nodes []*ast.Ident) []lsp.Locat
 
 func goRangeToLSPLocation(fset *token.FileSet, pos token.Pos, end token.Pos) lsp.Location {
 	return lsp.Location{
-		URI:   "file://" + fset.Position(pos).Filename,
```

Comment:
We also return triple slash on unix, since `fset.Position(pos).Filename` starts with `/`. I'm surprised we can't do this on windows.

How easy is it to get a windows environment I can SSH or RDP into? This change touches a lot of places and is the sort of thing that seems like it would break in future. I'd like to confirm it is necessary.

Relatedly, can't we change `vfs.OS` so it returns paths that just work?